dags/main_summary.py

```diff
@@ -598,3 +626,5 @@
 
 taar_locale_job.set_upstream(clients_daily_v6)
 taar_collaborative_recommender.set_upstream(clients_daily_v6)
+
+bgbb_pred.set_upstream(main_summary)
```

Comment:
@wcbeard should this actually be downstream of client's daily instead of main_summary?

Reply:
Oops. Yes, that's correct, it should be clients_daily.